### ISSUE TITLE
Test Last Log backup times: mistakes because of time zone differences

### ISSUE THREAD

**Issue by Geo-Ron** [sqlcollaborative/dbachecks#610]
### General Troubleshooting steps
- [ x ] Verified running the latest release of dbachecks?
- [ x ] Verified errors are not related to permissions?
- [ x ] Can duplicate in new/clean PowerShell session (clean = `powershell -NoProfile`)?

### Version Information
 - Operating System (Name|Version): Win 10 1803
 - PowerShell Version: Desktop 5.1.17134.590
 - SQL Server (Edition|Version): All

### Steps to Reproduce
The time zone at my SQL instance is Central Standard Time (GMT+1) 
The time zone at my workstation is Central European Time (GMT-5) 

- Run a transaction log backup and check if it is succesfull
- Invoke check:
`Invoke-DbcCheck -SqlInstance $instance.Fullname -ComputerName $Instance.Computername -Tags LastLogBackup`

The test results are:
```
Executing script C:\Program Files\WindowsPowerShell\Modules\dbachecks\1.1.171\checks\Database.Tests.ps1

  Describing Last Log Backup Times

    Context Testing last log backups on VRCVBSDB010
      [-] UtilityDB log backups on VRCVBSDB010 Should Be less than 15 minutes 7ms
        Expected the actual value to be greater than 2019-03-25T22:03:03.1999044+01:00, because Taking regular backups is extraordinarily important, but got 2019-03-25T16:10:02.0000000.
        494:                             $psitem.LastLogBackupDate | Should -BeGreaterThan (Get-Date).AddMinutes( - ($maxlog) + 1) -Because Taking regular backups is extraordinarily important
        at <ScriptBlock>, C:\Program Files\WindowsPowerShell\Modules\dbachecks\1.1.171\checks\Database.Tests.ps1: line 494
Tests completed in 2.08s
Tests Passed: 0, Failed: 1, Skipped: 0, Pending: 0, Inconclusive: 0
```

### Description of Bug

Due to the time zone differences between my workstation and the SQL server the test incorrectly reports it has been too long since the last log backup.

**Comment by stu-cross** [comment]
This still seems to be a problem. Running the tool on UTC, Server on Us East.

**Comment by conseilit** [comment]
Hi,

I also have the issue with SQL instances on other TZ than the computer running the PS Scripts.

I've updated the code to fit my needs :  

    Describe "Last Log Backup Times" -Tags LastLogBackup, LastBackup, Backup, DISA, Varied, $filename {
        $maxlog = Get-DbcConfigValue policy.backup.logmaxminutes
        $graceperiod = Get-DbcConfigValue policy.backup.newdbgraceperiod
        $skipreadonly = Get-DbcConfigValue skip.backup.readonly
        $skipsecondaries = Get-DbcConfigValue skip.backup.secondaries
		[DateTime]$sqlinstancedatetime = $InstanceSMO.Query("SELECT getutcdate() as getutcdate").getutcdate
		[DateTime]$oldestbackupdateallowed = $sqlinstancedatetime.AddHours( - $graceperiod)
        if ($NotContactable -contains $psitem) {
            Context "Testing last log backups on $psitem" {
                It "Can't Connect to $Psitem" {
                    $true | Should -BeFalse -Because "The instance should be available to be connected to!"
                }
            }
        }
        else {
            Context "Testing last log backups on $psitem" {
                @($InstanceSMO.Databases.Where{ (-not $psitem.IsSystemObject) -and $Psitem.CreateDate.ToUniversalTime() -lt $oldestbackupdateallowed -and $(if ($Database) { $PsItem.Name -in $Database }else { $ExcludedDatabases -notcontains $PsItem.Name }) }).ForEach{
                    if ($psitem.RecoveryModel -ne "Simple") {
                        if ($psitem.AvailabilityGroupName) {
                            $agReplicaRole = $InstanceSMO.AvailabilityGroups[$psitem.AvailabilityGroupName].LocalReplicaRole
                        } else {
                            $agReplicaRole = $null
                        }
                        $skip = ($psitem.Status -match "Offline") -or ($psitem.IsAccessible -eq $false) -or ($psitem.Readonly -eq $true -and $skipreadonly -eq $true) -or ($agReplicaRole -eq 'Secondary' -and $skipsecondaries -eq $true)
                        It -Skip:$skip  "Database $($psitem.Name) log backups should be less than $maxlog minutes old on $($psitem.Parent.Name)" {
                            $psitem.LastLogBackupDate | Should -BeGreaterThan $sqlinstancedatetime.AddMinutes( - ($maxlog) + 1) -Because "Taking regular backups is extraordinarily important"
                        }
                    }
                }
            }
        }
    }


Basically, I taking teh time onthe instance and avoid any call to ToUniversalTime().

Feel free to test.
Regards,
Christophe (@Conseilit)

**Comment by SQLDBAWithABeard** [comment]
Thats awesome, thank you for looking - could you create a PR with your updated code please
